molecule/model/schema_v2.py

```diff
@@ -710,6 +710,85 @@ def pre_validate_base_schema(env, keep_string):
     },
 }
 
+platforms_lxd_schema = {
+    'platforms': {
+        'type': 'list',
+        'schema': {
+            'type': 'dict',
+            'schema': {
+                'name': {
+                    'type': 'string',
+                },
+                'url': {
+                    'type': 'string',
+                },
+                'cert_file': {
+                    'type': 'string',
+                },
+                'key_file': {
+                    'type': 'string',
+                },
+                'trust_password': {
+                    'type': 'string',
+                },
+                'source': {
+                    'type': 'dict',
+                    'schema': {
+                        'type': {
+                            'type': 'string',
+                        },
+                        'mode': {
+                            'type': 'string',
+                            'allowed': [
+                                'pull',
+                                'local',
+                            ],
+                        },
+                        'server': {
+                            'type': 'string',
+                        },
+                        'protocol': {
+                            'type': 'string',
+                            'allowed': [
+                                'lxd',
+                                'simplestreams',
+                            ],
+                        },
+                        'alias': {
+                            'type': 'string',
+                        },
+                    },
+                },
+                'architecture': {
+                    'type': 'string',
+                    'allowed': [
+                        'x86_64',
+                        'i686',
+                    ],
+                },
+                'devices': {
```

Comment:
Please take a good look at the devices schema, that is the one I struggled with since I am not familiar with the markup. Might be not 100% correct